docs/solid-testing-library/api.mdx

```diff
@@ -0,0 +1,180 @@
+---
+id: api
+title: API
+sidebar_label: API
+---
+
+Due to being inspired by the [preact-testing-library][preact-docs] you can check
+its page for more information.
+
+[preact-docs]: ../preact-testing-library/intro.mdx
+
+There are several key differences, to be aware of.
+
+## Differences from the Preact Testing Library
```

Comment:
I don't know if this is needed.
Most Testing Libraries are all inspired by DOM testing library/each other.

Thoughts?

Reply:
I think that this doc is mainly showing comparisons (as can be seen in the `render` section, I'm still not sure it's needed though, I believe we can remove the "preact" examples and just keep the "solid" examples here.. Wdyt?

Reply:
done